### DISCUSSION TITLE
Jira 플랜 업그레이드는 필요가 없겠습니다.

### DISCUSSION THREAD

**Discussion by Woon-2** [MyeongWoonJang/cpp-study#12] [Ideas]
익명의 접근이 허용된다고 해서, 과거의 기록이 남는 것은 아닙니다.
매일매일 스크린샷을 남기는 것이 효율적일 것 같습니다.

![Roadmap](https://user-images.githubusercontent.com/73771162/191734674-8bc91e91-24b8-4ddf-a0fc-44c3bac4289b.png)
![Group 4](https://user-images.githubusercontent.com/73771162/191734691-037f83c6-7596-4f4b-a52d-78cb4a6ae9da.png)

이런 식으로 로드맵 한 장, 보드 한 장을 남기고
각 이슈 별로 하위 이슈 진척도 스샷 한 장씩 남기면 될 것 같습니다.
어떻습니까?

**Comment by Lcat12345** [comment]
제 생각에도 스크린샷을 남기는게 좋습니다.
대신 이슈 별로 하위 이슈 진척도를 본문의 스샷처럼 남긴다면 익명의 사람들에게는 눈에 잘 들어오지 않으니 별도의 부가 설명이 필요해 보입니다.
예를들면 텍스트로 Some Issue : 40% -> 50% 사진 아래에 써놓는것 처럼요.

**Reply by Woon-2** [reply]
부가 설명 좋습니다.
스크린샷 두 개로 비교를 하는 건 또 어떻습니까?

`Jira`의 유료 플랜 중에 왠지 project archiving이라는 단어도 보이고,
기록 남기는데 유용한 기능이 있을 것 같습니다.
프리미엄을 30일간 체험할 수 있는데, 지금 쓰기엔 우리가 너무 초짜다 보니 기간이 아깝고
나중에 본격적으로 써봐도 좋을 것 같습니다.

**Reply by Lcat12345** [reply]
스크린샷이 두개로 비교하는것도 좋겠네요. 방법은 여러가지로 나올수 있다고 생각합니다.

유료 플랜 체험도 나중에 해보죠
